.github/PULL_REQUEST_TEMPLATE/release.md

```diff
@@ -0,0 +1,77 @@
+# Release vFIXME
```

Comment:
Is there any prerequisites to being able to do these releases? e.g. "Make sure you are in X group and have Y permission."

Reply:
No, every datadog employee contributor of this repo (configured with our github team groups) is able to do these releases and steps.